infra/build_specified_commit_test.py

```diff
@@ -69,17 +69,17 @@ def test_detect_main_repo_from_commit(self):
     repo_origin, repo_name = build_specified_commit.detect_main_repo(
         'curl', commit='bc5d22c3dede2f04870c37aec9a50474c4b888ad')
     self.assertEqual(repo_origin, 'https://github.com/curl/curl.git')
-    self.assertEqual(repo_name, 'curl')
+    self.assertEqual(repo_name, '/src/curl')
```

Comment:
nit: rename all the repo_name vars to repo_path.

Reply:
Done.